jgrapht-guava/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jgrapht</groupId>
+		<artifactId>jgrapht</artifactId>
+		<version>1.1.1-SNAPSHOT</version>
+	</parent>
+	<artifactId>jgrapht-guava</artifactId>
+	<name>JGraphT - Guava Adapter</name>
+	<properties>
+		<main.basedir>${project.parent.basedir}</main.basedir>
+	</properties>
+	<licenses>
+		<license>
+			<name>GNU Lesser General Public License Version 2.1, February 1999</name>
+			<url>http://jgrapht.sourceforge.net/LGPL.html</url>
+			<distribution>repo</distribution>
+		</license>
+		<license>
+			<name>Eclipse Public License (EPL) 1.0</name>
+			<url>http://www.eclipse.org/legal/epl-v10.html</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<version>1.5</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<transformers>
+						<transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+							<manifestEntries>
+								<!-- Use unique OSGi bundle symbolic name for uber artifact. Otherwise 
+									e.g. jgrapht-guava-x.y.z.jar and jgrapht-guava-x.y.z-uber.jar could not be deployed 
+									to the same P2 repository. -->
+								<Bundle-SymbolicName>org.jgrapht.guava.uber</Bundle-SymbolicName>
+							</manifestEntries>
+						</transformer>
+						<transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+							<resource>META-INF/guava-license</resource>
+							<file>${main.basedir}/etc/licenses/apache-license-2.0.txt</file>
+						</transformer>
+						<transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+							<resource>META-INF/errorprone-license</resource>
+							<file>${main.basedir}/etc/licenses/apache-license-2.0.txt</file>
+						</transformer>
+						<transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+							<resource>META-INF/j2objc-license</resource>
+							<file>${main.basedir}/etc/licenses/apache-license-2.0.txt</file>
+						</transformer>
+						<transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+							<resource>META-INF/animal-shifter-annotations-license</resource>
+							<file>${main.basedir}/etc/licenses/MIT-license.txt</file>
```

Comment:
That copy of the license leaves the copyright line holder and year blank.  I think the best we can do is to use an extract from the pom.xml:

https://github.com/mojohaus/animal-sniffer/blob/master/animal-sniffer-annotations/pom.xml